reana_workflow_controller/consumer.py

```diff
@@ -76,8 +73,6 @@ def _update_workflow_status(workflow_uuid, status, logs):
         _update_commit_status(workflow, status)
     alive_statuses = \
```

Comment:
Can you delete this as well? It is not used either.

Reply:
Remember to rebase :)

Reply:
done :)